### PR TITLE
Remove upgrade actions from PackagesTable

### PR DESCRIPTION
**Pull request by jfurrow**

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @mlunoe

**Comment by kennyt** [comment]
can you write in the descriptions why this is happening? did people not want this on the table anymore?

**Comment by jfurrow** [comment]
@kennyt Honestly I'm not exactly sure why it's happening...

**Comment by kennyt** [comment]
@jfurrow what i mean was i didn't know if either it's being removed because it's never coming back, or if you're removing it because you're refactoring into another component. i'm assuming it's never coming back 👍

**Comment by jfurrow** [comment]
@kennyt Ahh right on, yeah I think either it will be implemented elsewhere, or maybe in this table, but in a future release.

**Comment by kennyt** [comment]
px

is the `Remove` button supposed to be there?

**Comment by kennyt** [comment]
:ship: after that right align comment

File: src/js/components/PackagesTable.js [hunk 1]
```diff
@@ -97,7 +82,7 @@ class PackagesTable extends React.Component {
         headerClassName: getClassName,
         heading: function () {},
         prop: 'uninstall',
-        render: this.getUpgradeCell,
```
Comment (mlunoe) [inline src/js/components/PackagesTable.js]: I think we need to decrease the column width for this column as well

Comment (jfurrow) [inline src/js/components/PackagesTable.js]: Done!

**Comment by jfurrow** [comment]
@kennyt It's right-aligned now :)

**Comment by kennyt** [comment]
tested this and it looks good 👍

**Pull request merged**